R/pkg/inst/tests/testthat/test_utils.R

```diff
@@ -232,9 +232,9 @@ test_that("basenameSansExtFromUrl", {
   x <- paste0("http://people.apache.org/~pwendell/spark-nightly/spark-branch-2.1-bin/spark-2.1.1-",
               "SNAPSHOT-2016_12_09_11_08-eb2d9bf-bin/spark-2.1.1-SNAPSHOT-bin-hadoop2.7.tgz")
   y <- paste0("http://people.apache.org/~pwendell/spark-releases/spark-2.1.0-rc2-bin/spark-2.1.0-",
-              "bin-hadoop2.4-without-hive.tgz")
+              "bin-hadoop2.6-without-hive.tgz")
```

Comment:
Actually there is no such artifact anymore. Just remove this line and the one below that also referenced it